src/components/MultiStepIndicator/components/VerticalStepIndicator/VerticalStepIndicator.scss

```diff
@@ -0,0 +1,180 @@
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 0px;
+
+  &.clickable {
+    cursor: pointer;
+  }
+
+  &--with-animation {
+    animation: circle-pop 250ms;
+  }
+
+  &--swap {
+    /* New state enters */
+    &-enter {
+      opacity: 0;
+      transform: translateY(15px);
+    }
+
+    &-enter-active {
+      position: relative;
+      opacity: 1;
+      transition: transform 150ms cubic-bezier(0, 0, 0.35, 1), opacity 70ms;
+      transform: translateY(0);
+    }
+
+    /* Old state leaves */
+    &-exit {
+      position: relative;
+      opacity: 1;
+      transform: translateY(0);
+    }
+
+    &-exit-active {
+      position: relative;
+      opacity: 0;
+      transform: translateY(-15px);
+      transition: transform 150ms cubic-bezier(0, 0, 0.35, 1), opacity 70ms;
+    }
+  }
+  &__number-divider-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+
+    .monday-style-divider {
+      @include theme-prop(background-color, ui-border-color);
+      margin: 16px 0 16px 16px;
+      height: 1px;
+      flex-grow: 1;
+    }
+  }
+
+  &__number-container {
+    width: 36px;
+    height: 36px;
+    border-radius: 50%;
+    display: flex;
+    flex-shrink: 0;
+
+    &__text {
+      margin: auto;
+      display: flex;
+      align-items: center;
+
+      &__check-icon {
+        width: 20px;
+        height: 20px;
+        margin-top: 2px;
+      }
+    }
+  }
+
+  &__text-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 8px;
+
+    &__title {
+      width: 100%;
+      .visually-hidden {
+        display: none;
+      }
+
+      &__text {
+        @include font-paragraph();
+        @include theme-prop(color, primary-text-color);
+      }
+    }
+
+    &__subtitle__text {
+      width: 100%;
+      font-size: $font-size-default;
+      @include theme-prop(color, disabled-text-color);
+    }
+  }
+
+  &--type {
+    &-primary {
+      &__number-container {
+        @include theme-prop(background-color, primary-color);
+      }
+    }
+    &-danger {
+      &__number-container {
+        @include theme-prop(background-color, negative-color);
+      }
+    }
+    &-success {
+      &__number-container {
+        @include theme-prop(background-color, positive-color);
+      }
+    }
+    &-dark {
+      &__number-container {
+        @include theme-prop(background-color, primary-text-color);
+      }
+    }
+  }
+
+  &--status {
+    &-pending {
+      &__number-container {
+        @include theme-prop(background-color, disabled-background-color);
+      }
+    }
+  }
+
+  @keyframes circle-pop {
+    0% {
+      transform: scale(1);
+    }
+
+    33% {
+      transform: scale(1.1);
+    }
+
+    66% {
+      transform: scale(0.97);
+    }
+
+    100% {
+      transform: scale(1);
+    }
+  }
+}
+
+// Fixes unreadability for light colored text inside active/fulfilled steps, in dark mode.
+.dark-app-theme {
```

Comment:
we have another theme and plan to add more, this fix should be in the color keys level and not component mode.
lets discuss this problem in greater depth please

Reply:
talked to Udi and this is an old fix that apparently is not needed anymore so I'm removing it